eclass/meson.eclass

```diff
@@ -285,6 +285,8 @@ meson_feature() {
 meson_src_configure() {
 	debug-print-function ${FUNCNAME} "$@"
 
+	[[ -n "${NINJA_DEPEND}" ]] || ewarn "Unknown value '${NINJA}' for \${NINJA}"
```

Comment:
I think we should just do this in global scope, within the case statement.

Reply:
I tried this at first, but it spammed the warning many times while resolving dependencies. I'm not sure there are any examples of `ewarn` used in the global scope without an explicit error?